src/config/dynamic-theme-fixes.config

```diff
@@ -8461,14 +8461,12 @@ INVERT
 iett.istanbul
 
 CSS
-.LineTimeDuyuru > li:nth-child(1) {
-    display:none;
+#routedesc p:nth-child(2) {
+    font-size: 0;                                                                                                                                                                                         
 }
-.LineTimeDuyuru::before {
-    content: "Kırmızı renkli seferler ÖHO ve OAŞ'a, beyaz renkli seferler İETT'ye aittir."
-}
-.DetailTable img {
-    filter: invert(100%) hue-rotate(180deg) !important;
+#routedesc p:nth-child(2)::after {
+    font-size: 20px;
+    content: "Kırmızı renkli seferler ÖHO ve OAŞ; beyaz renkli seferler İETT’ye aittir."
```

Comment:
Is it still not fixed?

Reply:
No, they still operate according to separate rules despite they are integrated to each other.